ompi/attribute/attribute.c

```diff
@@ -720,7 +797,7 @@ int ompi_attr_set_c(ompi_attribute_type_t type, void *object,
                     opal_hash_table_t **attr_hash,
                     int key, void *attribute, bool predefined)
 {
-    int ret;
+    int ret = MPI_SUCCESS;
```

Comment:
Same here. OMPI_SUCCESS for consistency. These functions should probably all work with internal error codes that are then translated to the MPI error space in the bindings.

Maybe we should split out all the changes to these files into a PR now. Would be easier to review. I can do that if you like. I have some time today to work on OMPI stuff.

Reply:
that would be great Nathan.  thanks!